cmd/builder/.goreleaser.yml

```diff
@@ -0,0 +1,39 @@
+before:
+  hooks:
+    - go mod download
+monorepo:
+  tag_prefix: cmd/builder/
+  dir: .core/cmd/builder
+builds:
+  - flags:
+      - -trimpath
+    ldflags:
+      - -s -w
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+      - ppc64le
+    ignore:
+      - goos: windows
+        goarch: arm64
+    binary: ocb
+release:
+  github:
+    owner: open-telemetry
+    name: opentelemetry-collector-releases
+  header: |
+    ### Images and binaries here: https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/{{ .Tag }}
```

Comment:
Wouldn't this just point to the current release?

Reply:
yes, it is a remnant from when they were released in the core repo, can probably remove this line

Reply:
In fact, it points to the main "vX.YYY.Z" release, and would be an easy way for someone to get from the cmd/builder/vX.YYY.Z to the main binaries release, so I think it's ok to leave it in. See link: https://github.com/jackgopack4/opentelemetry-collector-releases/releases/tag/cmd%2Fbuilder%2Fv0.106.1

Reply:
I would reword this to say something like "Collector images and binaries here", but that's fine with me.

Reply:
"Images and binaries for collector distributions here"